### ISSUE TITLE
Unclear wording in LCP audit

### ISSUE THREAD

**Issue by exterkamp** [GoogleChrome/lighthouse#10977]
I got some feedback from the translators that the LCP element audit is confusing and I agree.

The [current copy](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/largest-contentful-paint-element.js#L15) is:
```
This is the element that was identified as the Largest Contentful Paint.
```

However that is odd because an _element_ can't _be_ the "LCP". The element could _be_ the largest element painted and thus be the element identified as the element used to calculate LCP.

Possible reword:
```
This is the element that had the largest content during the load, therefore
its paint time was used in calculating the Largest Contentful Paint timing metric.
```

That might be too wordy, but I think we need to be more clear about what this element is.

**Comment by connorjclark** [comment]
`This is the element that is associated with the Largest Contentful Paint`.

**Comment by exterkamp** [comment]
Very succinct. SGTM 😄

**Comment by brendankenny** [comment]
Translator's note: "what do you mean by 'associated'" :P

**Comment by brendankenny** [comment]
"Largest Contentful Paint element" is the title immediately over the description, so it seems fine to leave the "LCP" part out and say something like

"This is the largest content element painted within the viewport."

<img width="896" alt="LCP description attempt" src="https://user-images.githubusercontent.com/316891/84824271-a3e49d00-afed-11ea-81b8-8726b9a20015.png">

**Comment by exterkamp** [comment]
Is "content element" a thing, or is something like this more natural:

"This is the largest element containing content painted within the viewport."

**Comment by patrickhulce** [comment]
That specific rewording feels more ambiguous to me.

i.e. is it 

`((largest element) containing content) painted within the viewport`

or

`(largest (element containing content)) painted within the viewport`

or

`largest (element containing (content painted within the viewport))`

I think the actual answer is none of our suggestions so far 😆 but "direct parent of text with largest visible area or image with largest visible area" doesn't exactly roll off the tongue... 

I wonder if we should head that direction though just to do more explaining of what in the world content element means?

**Comment by brendankenny** [comment]
ha, apparently didn't hit "Comment" on this:

> Is "content element" a thing

I just copied and pasted what was in https://web.dev/lcp/#what-is-lcp

Agreed it's not really a thing :)

> I wonder if we should head that direction though just to do more explaining of what in the world content element means?

+1 from me

**Comment by paulirish** [comment]
consensus. 

> "This is the largest contentful element painted within the viewport."
